src/views/v2/tickets/item/content.php

```diff
@@ -50,8 +50,16 @@
 	return;
 }
 
-$this->template( 'v2/tickets/item/content/title' );
+$context = $this->get_values();
 
-$this->template( 'v2/tickets/item/content/description' );
+$this->template(
+	'v2/tickets/item/content/title',
+	[
+		'ticket'  => $ticket,
+		'post_id' => $post_id,
+	]
+);
 
-$this->template( 'v2/tickets/item/extra' );
+$this->template( 'v2/tickets/item/content/description', $context );
+
+$this->template( 'v2/tickets/item/extra', $context );
```

Comment:
None of these changes should be necessary, if they are, let's save whatever the root cause is for a future PR to address it in common.